src/index.js

```diff
@@ -73,6 +74,9 @@ const routes = {
   }
 }
 
+// We will also configure the storage function that withRequest will use.
+configure({ storage: window.localStorage })
```

Comment:
I think it got lost in our conversation, but this was the only missing piece. It tells the underlying request function that <Form /> uses to check for the auth token in localStorage so you don't have to manually add the header.

Reply:
Excellent, thanks!